backend/main.py

```
@@ -104,6 +108,14 @@ async def lifespan(app: FastAPI):
 
 api = FastAPI(title="Devr.AI API", version="1.0", lifespan=lifespan)
 
+# Setup rate limiting
+limiter = get_limiter()
+api.state.limiter = limiter
+api.add_exception_handler(RateLimitExceeded, lambda req, e: {
```

Ruff (0.14.13)

Unused lambda argument: `req`

(ARG005)